AddPlugSpa/src/connect.ts

```diff
@@ -0,0 +1,40 @@
+async function onSubmit(e: Event) {
+  e.preventDefault();
+
+  const searchParams = new URLSearchParams(window.location.search);
+  const deviceId = searchParams.get('id');
+
+  let ssid = (document.getElementById('ssid') as HTMLInputElement).value;
+  let pass = (document.getElementById('pw') as HTMLInputElement).value;
+
+  await fetch('http://192.168.33.1/settings?allow_cross_origin=1').catch(); // Enable CORS
+  await fetch(
+    `http://192.168.33.1/settings?mqtt_enable=true
```

Comment:
hard-coded ip?

Reply:
This is OK, as per specification, the Shelly IP will always be 192.168.33.1 when connected to the Shelly AP